popUp.css

```diff
@@ -0,0 +1,197 @@
+/* pop-up styles */
+
+.popup-container {
+  display: none;
+  position: fixed;
+  z-index: 1;
+  overflow: auto;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: #c1c7d0e5;
+  backdrop-filter: blur(2px);
+}
+
+.main-pu-container {
+  margin: 40px 16px;
+  background-color: white;
+  border-radius: 16px;
+}
+
+.content-container {
+  display: flex;
+  flex-direction: column;
+  margin: 10px 10px;
+}
+
+.heading-btn {
+  display: flex;
+}
+
+.project-title-pu {
+  color: #172b4d;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  margin-bottom: 10px;
+}
+
+.close-btn-pu {
+  position: absolute;
+  right: 0;
+  top: 50px;
+  background: transparent;
+  cursor: pointer;
+  width: 40px;
+  height: 50px;
+  border: none;
+  font-weight: lighter;
+  font-size: 20px;
+  margin-right: 20px;
+}
+
+.devs-pu {
+  display: flex;
+  list-style-type: none;
+  gap: 10px;
+}
+
+.dev-pu {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.dev-pu2 {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.project-img-pu {
+  width: 100%;
+  height: 100%;
+  margin-top: 15px;
+}
+
+.project-info-pu {
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  margin-top: 10px;
+}
+
+.tags-pu {
+  display: flex;
+  list-style-type: none;
+  align-items: baseline;
+  color: #6070ff;
+}
+
+.tag-pu {
+  font-size: 15px;
+  background-color: #ebebff;
+  width: 80px;
+}
```

Comment:
- [ ] Your popup must follow the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, tests, margins,...) Kindly add margin-left and margin-right to each language or technology(HTML, CSS, javascript...), and don't forget to add a border-radius CSS property to each of it 🎨 🖌️ 

- [ ] Let's add the thinner transparent bar to separate the list of technologies with the `View Live and View Source` button 😄 

- [ ] The `See live and See Source` buttons must follow the original design, kindly remove the background color as you can see on the Figma design below 👇🏻 , we have a while background color 👍🏻 😜 

- Figma Design:

![button1](https://user-images.githubusercontent.com/79058364/173584878-5c6a8566-d03c-4c49-bebc-5741d53fad18.png)


- Your Design:

![23dest](https://user-images.githubusercontent.com/79058364/175261498-e19d4769-3556-455a-a32e-04c0dbcf0d8a.png)